src/rules/no-use-before-define.md

```diff
@@ -4,16 +4,15 @@ layout: doc
 rule_type: problem
 ---
 
+在 JavaScript 中，在 ES6 之前，变量和函数的声明被吊在一个作用域的顶端，所以有可能在代码中正式声明之前使用标识符。这可能会引起混淆，有些人认为最好是在使用变量和函数之前先声明它们。
 
-In JavaScript, prior to ES6, variable and function declarations are hoisted to the top of a scope, so it's possible to use identifiers before their formal declarations in code. This can be confusing and some believe it is best to always declare variables and functions before using them.
+在 ES6 中，块级绑定（`let` 和 `const`）引入了“temporal dead zone”，任何试图在变量声明之前访问它的行为都会被抛出一个 `ReferenceError`。
```

Comment:
```suggestion
在 ES6 中，块级绑定（`let` 和 `const`）引入了 "temporal dead zone"，任何试图在变量声明之前访问它的行为都会被抛出一个 `ReferenceError`。
```

Reply:
There is no problem here, this is double quotation marks in Chinese.